src/test/daml/Daml/Finance/Settlement/Test/Transfer.daml

```diff
@@ -316,7 +326,84 @@ run4 = script do
 
   pure ()
 
+-- Transfer of commercial bank money from Alice to Charlie via Bob (acting as an intermediary/passthrough).
+-- +---------------------------------------------------+
+-- | Accounts                                          |
+-- +---------------------------+-----------------------+
+-- | structure:                | used for:             |
+-- +---------------------------+-----------------------+
+-- |           Bank            |                       |
+-- |      /     |    \         | commercial bank money |
+-- | Sender -> CCP -> Receiver |                       |
+-- +---------------------------+-----------------------+
+run5 : Script ()
+run5 = script do
+  TestParties{..} <- setupParties
+
+  -- Account and holding factory
+  let fp = [("FactoryProvider", singleton publicParty)]
+  accountFactoryCid <- toInterfaceContractId <$> Account.createFactory provider fp
+  holdingFactoryCid <- toInterfaceContractId <$> submitMulti [publicParty] [] do
+    createCmd Fungible.Factory with provider = publicParty; observers = M.fromList fp
+
+  -- Create accounts
+  [senderAccount, ccpAccount, receiverAccount] <- mapA (Account.createAccount "Cash Account" [publicParty] accountFactoryCid holdingFactoryCid [] Account.Owner bank) [sender, ccp, receiver]
+
+  -- Distribute asset
+  now <- getTime
+  cashInstrument <- Instrument.originate cb cb "USD" "United States Dollar" [] now
+  transferableCid <- coerceContractId <$> Account.credit [publicParty] cashInstrument 200_000.0 senderAccount
+
+  -- Create settlement factory
+  settlementFactoryCid <- toInterfaceContractId @Factory.I <$> submitMulti [bank] [] do createCmd Factory with provider = bank; observers = empty
+
+  -- Instruct transfer
+  let
+    step1 = Step with sender; receiver = ccp; quantity = Instrument.qty 200_000.0 cashInstrument
+    step2 = Step with sender = ccp; receiver; quantity = Instrument.qty 200_000.0 cashInstrument
+  (batchCid, [instructionCid2, instructionCid1]) <- submitMulti [bank] [] do
+    exerciseCmd settlementFactoryCid Factory.Instruct with
+      instructors = singleton bank
+      settlers = singleton settler
+      id = Id "transfer 1"
+      description = "transfer of 200000.0 USD payment"
+      contextId = None
+      steps = [step2, step1] -- order does not matter
+
+  -- Approve instruction
+  instructionCid2 <- submitMulti [receiver] [publicParty] do
+    exerciseCmd instructionCid2 Instruction.Approve with
+      actors = singleton receiver
+      approval = TakeDelivery receiverAccount
+
+  -- Approve with passthrough
+  instructionCid1 <- submitMulti [ccp] [publicParty] do
+    exerciseCmd instructionCid1 Instruction.Approve with
+      actors = singleton ccp
+      approval = PassthroughTo (ccpAccount, InstructionKey with requestors = singleton bank; batchId = Id "transfer 1"; id = Id (show 1))
```

Comment:
```suggestion
      approval = PassthroughTo (ccpAccount, InstructionKey with requestors = singleton bank; batchId = Id "transfer 1"; id = Id "1")
```

😅

Reply:
don't need to change it, just found it funny

Reply:
I find it funny to:-)